VWOSdk/Core/VWOClient.cs

```diff
@@ -624,6 +627,71 @@ public bool FlushEvents()
                     LogInfoMessage.NoDataUserStorageService(file, campaignKey, userId);
                 }
             }
+
+            // Mutually Exclusive Group
+            IDictionary<string, dynamic> groupDetails = CampaignHelper.isPartOfGroup(this._settings, campaign.Id);
+            if (groupDetails.ContainsKey("groupId"))
+            {
+                dynamic groupId = 0, groupName = "";
+                if (groupDetails.Count > 0)

```

Comment:
isn't groupDetails count will be greater than always if :-  `if (groupDetails.ContainsKey("groupId")) ` is true ?

Reply:
you are right . fixed that